test/SILGen/shared.swift

```diff
@@ -1,4 +1,153 @@
 // RUN: %target-swift-frontend -Xllvm -sil-full-demangle -emit-silgen -sil-serialize-witness-tables %s -disable-objc-attr-requires-foundation-module | %FileCheck %s
```

Comment:
A couple of questions:

1. Why do you need to use stdlib types? I think it would be clearer what you are testing if you didn't use string for instance.
2. You don't have the ownership verifier enabled here. Do you run into false positives? To enable it you need to pass in the flag -enable-sil-ownership.